WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -882,6 +882,34 @@ extension ShippingLabelFormViewModel {
     }
 }
 
+// MARK: - Shipping Notice dismiss state handling
+extension ShippingLabelFormViewModel {
+    func setEUShippingNoticeDismissState(isDismissed: Bool,
+                                         onCompletion: @escaping (Bool) -> Void) {
```

Comment:
Since we're only using this to dismiss the notice (we never set the notice dismiss state to `false`) what about making this method something like `dismissEUShippingNotice(onCompletion: @escaping (Bool) -> Void)`? Then, we can call the action `AppSettingsAction.setEUShippingNoticeDismissState(isDismissed: true)`.

Reply:
I don't even know why I went with this parameter. Completely useless 😅 Thank you so much for noticing it! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/9695/commits/6ade9b4cd7177045e76f2b8659d1e790e18af4eb